samples/object_in_fluid/unit-test.py

```diff
@@ -0,0 +1,70 @@
+import espressomd
+
+from espressomd import lb
+from espressomd import lbboundaries
+from espressomd import shapes
+from espressomd.shapes import Rhomboid
+from espressomd.shapes import Cylinder
+from espressomd.interactions import OifLocalForces
+from espressomd.interactions import OifGlobalForces
+from espressomd.interactions import OifOutDirection
+
+import numpy as np
+import os, sys
+
+from object_in_fluid import *
```

Comment:
a wildcard import is not a good choice due to namespace pollution and readability, maybe sth. like the following would be appropriate:
```python
import object_in_fluid as oif
```